src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,45 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-12#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .row
+      .col-md-4.order-md-2.order-sm-1.mb-4#side-links
+        - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+          %h4
+            Reviews
+          .mb-4
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: accepted_reviews, as: :review
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: declined_reviews, as: :review
+            = render partial: 'webui/request/beta_show_tabs/review_summary', collection: open_reviews, as: :review
 
-  .comment_new.mt-3
-    = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+        - if open_reviews_for_staging_projects.present?
+          %h4
+            Reviews for staging projects
+          .mb-4
+            - open_reviews_for_staging_projects.each do |review|
```

Comment:
Not that we should hide this but there only ever will be one...

Reply:
We changed that part as we explained in the previews comment: https://github.com/openSUSE/open-build-service/pull/12863#discussion_r942365293